packages/react/src/primitives/SearchField/SearchField.tsx

```diff
@@ -78,7 +78,6 @@ const SearchFieldPrimitive: Primitive<SearchFieldProps, 'input'> = (
       innerEndComponent={
         <FieldClearButton
           ariaLabel={clearButtonLabel}
-          excludeFromTabOrder
```

Comment:
Can we add an a11y unit test asserting behavior that ensures that `excludeFromTabOrder` is not added back in the future?

Reply:
Yes! I have a test written for this now but currently blocked from running tests on cloud desktop due to GLIBC_2.28 not being present in AL2 images.

Reply:
Unit test verified via @calebpollman due to inability to run locally